src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/DoNotIgnoreMethodResultsTests.cs

```diff
@@ -418,10 +485,22 @@ public void ThrowsException()
             return;
         }});
     }}
-}}", useXunit ? XunitApis.CSharp : NUnitApis.CSharp },
-                GetCSharpObjectCreationResultAt(10, 13, "ThrowsException", "Test"));
-
-            VerifyBasic(new[] { $@"
+}}", useXunit ? XunitApis.CSharp : NUnitApis.CSharp
```

Comment:
📝 We should file a bug to replace all these with package references once dotnet/roslyn-sdk#329 is available

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/2997